src/number/bigIntFromString.ts

```diff
@@ -0,0 +1,126 @@
+/**
+ * Represents bigints converted from strings
```

Comment:
:thought_balloon:  `bigIntFromString` seems too verbose; I feel like this should just be `bigint` and it's implied it decodes/encodes from/to a string since we can't represent `BigInt`s outside JS any other way

Reply:
also the whole `from` naming puts too much emphasis on the `Decoder` instance in particular. but I don't have a better idea for the existing `intFromString` and `floatFromString`

Reply:
I think I'd prefer to keep the name as bigIntFromString, mostly because technically bigInt is a valid schema, it's just really boring.  And also [io-ts-types](https://github.com/gcanti/io-ts-types/blob/master/src/BigIntFromString.ts) calls it BigIntFromString.

And we might also want to add the other missing schema(bles) in io-ts-types too that seem useful, like unix time, etc